recipes/oommf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: oommf
+  version: "1.2b0_20160930b1"  # [unix]
+  version: "1.2b0_20160930b0"  # [win]
+
+# on Windows, we install from precompiled binary packages
+source:  # [unix]
+  git_url: https://github.com/fangohr/oommf  # [unix]
+  git_rev: 1.2b0_20160930b1  # [unix]
```

Comment:
See http://conda-forge.github.io/docs/meta.html#build-from-tarballs-not-repos

Use
```yaml
source:
   url: https://github.com/fangohr/oommf/archive/1.2b0_20160930b1.tar.gz  # [unix]
   url: http://math.nist.gov/oommf/dist/oommf12b0_20160930_86_x64.zip     # [win]
   sha256: <unixsha256>  # [unix]
   sha256: <windowssha256>  # [win]
```